src/main/java/core/basesyntax/FindNumbers.java

```diff
@@ -11,6 +14,21 @@
      * числа 92, 18, 26 и 0.</p>
      */
     public int[] getAllNumbers(String text) {
-        return null;
+        String[] temp = text.split("[^\\d-]");
+
+        List<Integer> notEmptyStrings = new ArrayList<>();
```

Comment:
see checklist - `Let’s use method replaceAll() with regular expression, and method split() in this task.`

and let's not use List here, only arrays